flask_api.py

```diff
@@ -108,8 +109,14 @@ def handle_question():
     if "question" not in request_body:
         return "request body should include the question", BAD_REQUEST
 
-    response = {"answer": nimbus.answer_question(question)}
+    # Store
+    db = NimbusMySQLAlchemy(config_file=CONFIG_FILE_PATH)
```

Comment:
This line needs to be replaced with `db = initiableDB()`, I think. Everything else looks fine to me

Reply:
On it